src/App.js

```diff
@@ -85,45 +79,47 @@ function ChatRoom() {
       text: formValue,
       createdAt: firebase.firestore.FieldValue.serverTimestamp(),
       uid,
-      photoURL
-    })
+      photoURL,
+    });
 
     setFormValue('');
     dummy.current.scrollIntoView({ behavior: 'smooth' });
-  }
+  };
 
-  return (<>
-    <main>
+  return (
+    <>
+      <main>
 
-      {messages && messages.map(msg => <ChatMessage key={msg.id} message={msg} />)}
+        {messages && messages.map((msg) => <ChatMessage key={msg.id} message={msg} />)}
 
-      <span ref={dummy}></span>
+        <span ref={dummy} />
 
-    </main>
+      </main>
 
-    <form onSubmit={sendMessage}>
+      <form onSubmit={sendMessage}>
 
-      <input value={formValue} onChange={(e) => setFormValue(e.target.value)} placeholder="say something nice" />
+        <input value={formValue} onChange={(e) => setFormValue(e.target.value)} placeholder="say something nice" />
 
-      <button type="submit" disabled={!formValue}>🕊️</button>
+        <button type="submit" disabled={!formValue}>🕊️</button>
```

Comment:
when I run this to my local, it has a warning to put a span, role, and aria-label. You can add it to compiled it successfully!